validation/lagom-validation/user-management.md

```diff
@@ -0,0 +1,109 @@
+# User Validation
+Every invalid equivalence class returns an error. Every valid class returns an empty sequence.
+
+## Address
+
+### Street (String)
+*validCharactersStreet* := Letters ∪ Umlauts ∪ Digits ∪ Whitespace ∪ {ß , .  -}
```

Comment:
What happens, if we have international students that have their erstwohnsitz in e.g. China but can digitally study in germany? They would have some chinese symbols.

Reply:
While I agree that is the case, the international names of these streets/cities etc need to suffice for now